proto/tendermint/abci/types.proto

```diff
@@ -132,6 +132,9 @@ message RequestPrepareProposal {
   repeated bytes block_data = 1;
   // If an application decides to populate block_data with extra information, they can not exceed this value.
   int64 block_data_size = 2;
+  // votes includes all votes from the previous block. This contains vote extension data that can be used in proposal
+  // preparation. The votes here will then form the last commit that gets sent in the proposed block.
```

Comment:
I'm not sure if it needs to be done in this PR but this comment shouldn't mention vote extensions

Reply:
Yes, this is dealt with in the upcoming PR with the backport of #8094